src/test/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockitoTest.java

```diff
@@ -0,0 +1,667 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.jmockit;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+import org.openrewrite.test.SourceSpec;
+import org.openrewrite.test.TypeValidation;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.java.testing.jmockit.JMockitTestUtils.*;
+
+class JMockitMockUpToMockitoTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        setParserSettings(spec, JMOCKIT_DEPENDENCY, JUNIT_4_DEPENDENCY);
+    }
+
+    @DocumentExample
+    @Test
+    void mockUpStaticMethodTest() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import mockit.Mock;
+              import mockit.MockUp;
+              import static org.junit.Assert.assertEquals;
+              import org.junit.Test;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      new MockUp<MyClazz>() {
+              
+                          @Mock
+                          public int staticMethod() {
+                              return 1024;
+                          }
+              
+                          @Mock
+                          public int staticMethod(int v) {
+                              return 128;
+                          }
+                      };
+                      assertEquals(1024, MyClazz.staticMethod());
+                      assertEquals(128, MyClazz.staticMethod(0));
+                  }
+              
+                  public static class MyClazz {
+                      public static int staticMethod() {
+                          return 0;
+                      }
+              
+                      public static int staticMethod(int v) {
+                          return 1;
+                      }
+                  }
+              }
+              """, """
+              import static org.junit.Assert.assertEquals;
+              import static org.mockito.ArgumentMatchers.*;
+              import static org.mockito.Mockito.*;
+              
+              import org.junit.Test;
+              import org.mockito.MockedStatic;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      try (MockedStatic mockStaticMyClazz = mockStatic(MyClazz.class)) {
+                          mockStaticMyClazz.when(() -> MyClazz.staticMethod()).thenAnswer(invocation -> {
+                              return 1024;
```

Comment:
This line generates warning in Intellij "Statement lamdba can be replaced with expression lamdba". I'd prefer for migration to not generate additional warnings. To eliminate this, can either get rid of the parentheses, or can use thenReturn (which may be better to be done in separate PR as discussed). Regarding removal of parantheses, can be done in the code, or if too difficult, can run this recipe in doAfterVisit https://docs.openrewrite.org/recipes/staticanalysis/lambdablocktoexpression